backend/fms_core/models/freezeman_user.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class FreezemanUser(models.Model):
+    user = models.ForeignKey(User, on_delete=models.PROTECT, related_name='freezeman_user')
+
+    @property
+    def username(self):
+        return self.user.username
+
+    def __str__(self):
+        return "FreezemanUser{username=%}" % self.username
```

Comment:
Don't use the internal model name in the str descriptor ... this could be shown to the user ... since username is unique you can just display that. The rest of the str might clash with context specific uses.